### PR TITLE
Cailey/artifactory/directory fixes

### PR DESCRIPTION
**Pull request by caggles**
This PR updates the volumes for artifactory to be appropriate sizes, and updates the installation and vars files to reflect that.

Also I caught an issue with the storage config not getting picked up and applied properly, so there are some fixes in here relating to that too.

### CODE REVIEW

File: apps/artifactory/artifactory-ha/install.yaml [hunk 1]
```diff
@@ -126,6 +123,7 @@
                                              -p "IMAGE_STREAM_TAG=patroni:v{{ db_version }}-stable" \
                                              -p IMAGE_REGISTRY={{ image_registry }} \
                                              -p STORAGE_CLASS={{ db_storage_class }} \
+                                             -p PVC_SIZE={{ db_storage_size }} \
```
Comment (ShellyXueHan) [inline apps/artifactory/artifactory-ha/install.yaml:126]: is this param in the db template?

**Review by ShellyXueHan — APPROVED** [review]

**Review by ShellyXueHan — APPROVED** [review]

**Review by jefkel — CHANGES_REQUESTED** [review]
Getting there, storage fixes look fine, documentation updates are good to keep in there (feels like some of those documentation notes should be listed in tickets to track/resolve outstanding work)
Things to keep in mind (as I see this change includes updating the remote repository)
- We don't want to be pulling from a remote repository in our manifests.  Image needs to be imported during a pipeline stage (or currently added to ansible deploy as a separate step?), and then statefulset configurations need to use the local cluster image instead.

- Recommend against a repository path being used for certs/license files.  (not in gitignore, so possibility exists of pushing secrets to repo.)  Could use an environment variable to reference the external location (or just hard-code a ~/project-private/certs/ etc structure that you can create anywhere?)

**Comment by caggles** [comment]
> Getting there, storage fixes look fine, documentation updates are good to keep in there (feels like some of those documentation notes should be listed in tickets to track/resolve outstanding work)
> Things to keep in mind (as I see this change includes updating the remote repository)
> 
> * We don't want to be pulling from a remote repository in our manifests.  Image needs to be imported during a pipeline stage (or currently added to ansible deploy as a separate step?), and then statefulset configurations need to use the local cluster image instead.
> * Recommend against a repository path being used for certs/license files.  (not in gitignore, so possibility exists of pushing secrets to repo.)  Could use an environment variable to reference the external location (or just hard-code a ~/project-private/certs/ etc structure that you can create anywhere?)

done and done :D

**Review by jefkel — APPROVED** [review]
check and check

**Pull request merged**